dotnet/src/Connectors/Connectors.OpenAI/OpenAIServiceCollectionExtensions.cs

```diff
@@ -1120,12 +1123,11 @@ public static class OpenAIServiceCollectionExtensions
 
```

Comment:
Given we pass in a modelId we  should pass it to the service and have set it in the service attributes and it will be used by the service selector

Reply:
@markwallace-microsoft I followed Dmytro suggestion of removing the ModelId, not sure if this comment applies anymore.

Reply:
Brought back the ModeId